### PR TITLE
Update EIP-1753: Move to draft

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-1753.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ created: 2019-02-06
 
 ## Abstract
 
-This Ethereum Improvement Proposal (EIP) proposes an Ethereum standard for the issuance of licences, permits and grants (Licences). 
+This Ethereum Improvement Proposal (EIP) proposes an Ethereum standard for the issuance of licences, permits and grants (Licences) as a Non Transferable Token NTT.
```
Comment (xinbenlv) [inline EIPS/eip-1753.md:13]: Hi @bitcoinbrisbane , welcome back to EIP process. This EIP has been Stagnant for a while, if you intend to revive this EIP and resume the process to move forward, can you update the status back to Draft and then fix all linter errors?
```suggestion
This Ethereum Improvement Proposal (EIP) proposes an Ethereum standard for the issuance of licences, permits and grants (Licences) as a Non Transferable Token (NTT). 
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Please fix the walidator issues

File: EIPS/eip-1753.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
 eip: 1753
 title: Smart Contract Interface for Licences
 author: Lucas Cullen (@BitcoinBrisbane), Kai Yeung (@CivicKai), Anna Crowley <annaelizabethcrowley@gmail.com>, Caroline Marshall <caroline.marshall888@gmail.com>, Katrina Donaghy <katrina@civicledger.com>
-status: Stagnant
+status: Review
```
Comment (Pandapip1) [inline EIPS/eip-1753.md:5]:
```suggestion
status: Draft
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Please also fix the EIPW errors

**Pull request closed without merging**